x-pack/plugins/spaces/public/views/management/_manage_spaces.scss

```diff
@@ -1,3 +1,5 @@
+@import 'x-pack/public/components/collapsible_panel/collapsible_panel';
```

Comment:
It's unnecessary to import other SASS files because all SASS files (including all in X-pack) get compiled and loaded on all pages, so importing again will duplicate the styles.